js/index.js

```diff
@@ -2,7 +2,13 @@ const answer = document.querySelector('[data-js="quizcard-answer"]');
 const answerButton = document.querySelector(
   '[data-js="quizcard-answer-button"]'
 );
+const button = document.querySelector('[data-js="buttonDarkMode"');
+const darkmode = document.querySelector('[data-js="bodyDarkMode"]');
 
 answerButton.addEventListener("click", () => {
   answer.classList.toggle("revealAnswer");
 });
+
+button.addEventListener("click", () => {
+  darkmode.classList.toggle("dark-mode");
+});
```

Comment:
Vermutung zum Dark Mode: 
querySelector gibt dir immer das erste (und nur das erste) Element aus dem Dokument zurück, wenn ich mich da richtig erinnere. Also sprichst du auf der index.html den body an und das ist prima. Auf den weiteren Seiten gehts dann aber nicht mehr, du bräuchtest also unique identifies für body-bookmarks und body-profile. Vom Prinzip her kein Problem, allerdings müsstest du dann auf mehrere .js-Dateien ausweichen. Grund: wenn du zB in der Bookmarks-Seite bist und die mit deiner index.js verknüpft ist, in der ein eventListener für ein einen Button vorhanden ist, der auf dieser Bookmarks-Seite gar nicht gefunden werden kann (ist ja in der index.html), dann hängt sich alles auf. Ich probiere es gleich mal aus. Alternativ hatte Marvin eine super Lösung mit localstorage, wo du nur einen Button brauchst, um alle Seiten in den Dark Mode zu versetzen, aber das übersteigt meine Kenntnisse :)

Reply:
jau, mit 3 .js-Dateien funktioniert es :) aufgepasst allerdings, die Bude ist case sensitive, ich marker dir gleich noch was in der bookmarks.html an ...